selftest/selftest_test.go

```diff
@@ -67,3 +75,45 @@ func (s *selftestSuite) TestRunNotHappy(c *C) {
 	c.Check(err, IsNil)
 	c.Check(unhappyCheckRan, Equals, 1)
 }
+
+func (s *selftestSuite) TestUnexportedChecks(c *C) {
+
+	// collect what funcs we run in selftest.Check
+	var runCheckers []string
+	v := reflect.ValueOf(selftest.Checks)
+	for i := 0; i < v.Len(); i++ {
+		v := v.Index(i)
+		fname := runtime.FuncForPC(v.Pointer()).Name()
+		l := strings.Split(fname, ".")
```

Comment:
Are we getting the last element of a sequence of dotted elements?

Reply:
Yes, I updated it to the suggestion from @bboozzoo